mne/viz/_3d.py

```diff
@@ -2521,7 +2523,12 @@ def _plot_stc(
     if overlay_alpha == 0:
         smoothing_steps = 1  # Disable smoothing to save time.
 
-    title = subject if len(hemis) > 1 else f"{subject} - {hemis[0]}"
+    sub_info = subject if len(hemis) > 1 else f"{subject} - {hemis[0]}"
+    if not title:
+        title = sub_info
+    else:
+        title = f"{title}, {sub_info}"
```

Comment:
I don't think this is the right behavior.  I don't like that there is no way to prevent subject name from appearing. One concrete case where you *don't* want to show it: if your subject names aren't de-identified, and you want to show an STC plot in a public presentation/demo.

Most (all?) other places in our codebase, if we have some sort of default/automatic title when `title=None`, then passing `title="whatever"` would *completely replace* the default title. I think we should do likewise here.
```suggestion
    title = title if title is not None else sub_info
```